index.html

```diff
@@ -8,20 +8,33 @@
     </head>
     <body class="overflow-hidden">
```

Comment:
```suggestion
    <body class="overflow-hidden flex flex-col min-h-screen">
```
In conjunction with the other suggestions below, this would ensure that the content uses the full screen if the proposed suggestions were to be accepted.

Reply:
Made a commit bbbbb7d2e0a8194acb7b147bb3725d98a0e2d2d5.